server/src/api/modules/groups/services/EditGroupService.ts

```diff
@@ -448,3 +479,20 @@ function calculateRoleChangeMaps(
 }
 
 export { editGroup };
+
+async function updateGroupRoleOrder(
+  groupId: number,
+  roleOrders: Array<{ role: GroupRole; index: number }>,
+  transaction: PrismaTypes.TransactionClient
+) {
+  // TODO: unsure if there's a better way to delete/replace records using prisma
```

Comment:
Would [`upsert`](https://www.prisma.io/docs/orm/reference/prisma-client-reference#upsert) work here?

Reply:
thought about it but unsure how it works with prisma, would it look at the compound key to do it since there isn't explicit ids to do it on.